QA/PoroDevTest/src/main/java/common/ui_setup/DriverSetup.java

```diff
@@ -1,4 +1,60 @@
 package common.ui_setup;
 
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import java.time.Duration;
+
 public class DriverSetup {
+    private WebDriver driver;
+    private int pageLoadTime;
+    private int elementDetectionTimeout;
+
+    public DriverSetup(DriverType browserType) {
+        pageLoadTime = 10;
```

Comment:
Initialization of pageLoad and timeout can be inside constants class

Reply:
Constant class is created